prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.krishna-test.yaml

```diff
@@ -42,9 +42,8 @@ presubmits:
 
   istio-releases/daily-release:
```

Comment:
is this supposed to be istio-releases/pipeline?

Reply:
yes